packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker-task-base-types.ts

```diff
@@ -55,31 +63,43 @@ export interface Channel {
 
     /**
      * Compression type if training data is compressed
+     *
+     * @default - None
      */
     readonly compressionType?: CompressionType;
 
     /**
-     * Content type
+     * The MIME type of the data.
+     *
+     * @default - None
      */
     readonly contentType?: string;
 
     /**
-     * Location of the data channel
+     * Location of the channel data.
```

Comment:
It should be `data channel`, no?

Reply:
I pulled it from the SageMaker [documentation](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_Channel.html)